cluster-autoscaler/simulator/clustersnapshot/basic.go

```diff
@@ -164,7 +164,7 @@ func (data *internalBasicSnapshotData) removeNode(nodeName string) error {
 	return nil
 }
 
-func (data *internalBasicSnapshotData) addPod(pod *apiv1.Pod, nodeName string) error {
+func (data *internalBasicSnapshotData) schedulePod(pod *apiv1.Pod, nodeName string) error {
```

Comment:
are these name changes really meaningful, given that we are simply wrapping the k/k scheduler's NodeInfo methods which will have the existing names?

Reply:
Yeah, the point is that when we introduce the DRA logic this will no longer be just a wrapper around the `schedulerframework.NodeInfo.AddPod`. There will be additional DRA processing, as well as interacting with the scheduler framework plugins.

I just want to get the interface names changed in one go to minimize conflicts later.

Reply:
FYI: I changed things a bit here - instead of renaming `AddPod` to `SchedulePod`, it's now renamed to `ForceAddPod`. I explained why in the relevant commit message.

Reply:
ACK, sgtm